libevmasm/Instruction.cpp

```diff
@@ -356,13 +356,13 @@ string dev::eth::disassemble(bytes const& _mem)
 	stringstream ret;
 	eachInstruction(_mem, [&](Instruction _instr, u256 const& _data) {
 		if (!isValidInstruction(_instr))
-			ret << "0x" << hex << int(_instr) << " ";
+			ret << "0x" << uppercase << hex << int(_instr) << " ";
```

Comment:
I know the `std` namespace is imported here, but I find the explicit prefix helpful for `uppercase` and `hex`, etc.

Reply:
Added the `std` for you. I kinda felt the same, but I usually try to keep the original code as much as possible when doing changes ;)